core/server/data/migrations/versions/3.18/02-add-members_stripe_connect-auth-permissions.js

```diff
@@ -0,0 +1,56 @@
+const ObjectId = require('bson-objectid');
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+    async up(options) {
+        const connection = options.transacting;
+
+        const existingIdentityPermission = await connection('permissions').where({
+            action_type: 'auth',
+            object_type: 'members_stripe_connect'
+        }).first();
+
+        if (existingIdentityPermission) {
+            logging.warn('Permission for auth:members_stripe_connect already added');
+            return;
+        }
+
+        logging.info('Adding permission for auth:members_stripe_connect');
+
+        const date = connection.raw('CURRENT_TIMESTAMP');
+
+        await connection('permissions').insert({
+            id: ObjectId.generate(),
+            name: 'Auth Stripe Connect for Members',
+            action_type: 'auth',
+            object_type: 'members_stripe_connect',
+            created_at: date,
+            created_by: 1,
+            updated_at: date,
+            updated_by: 1
+        });
```

Comment:
Is there a reason this isn't using the existing permissions fixtures?

Reply:
Yes! Because they go via the model layer, and I don't think we should be using the model layer for migrations anymore!

Reply:
👍 OK. Would be good to make a bit of noise about this so that everyone is aware we want to change patterns in migrations